x-pack/plugins/security_solution/public/overview/pages/data_quality.tsx

```diff
@@ -0,0 +1,254 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CommentType } from '@kbn/cases-plugin/common';
+import {
+  DataQualityPanel,
+  DATA_QUALITY_SUBTITLE,
+  ECS_REFERENCE_URL,
+  getIlmPhaseDescription,
+  ILM_PHASE,
+  INDEX_LIFECYCLE_MANAGEMENT_PHASES,
+  SELECT_ONE_OR_MORE_ILM_PHASES,
+} from '@kbn/ecs-data-quality-dashboard';
+import type { EuiComboBoxOptionOption } from '@elastic/eui';
+import {
+  EuiComboBox,
+  EuiFormControlLayout,
+  EuiFormLabel,
+  EuiLink,
+  EuiLoadingSpinner,
+  EuiText,
+  EuiToolTip,
+  useGeneratedHtmlId,
+} from '@elastic/eui';
+import React, { useCallback, useMemo, useState } from 'react';
+import styled from 'styled-components';
+
+import { SecurityPageName } from '../../app/types';
+import { getGroupByFieldsOnClick } from '../../common/components/alerts_treemap/lib/helpers';
+import { useTheme } from '../../common/components/charts/common';
+import { HeaderPage } from '../../common/components/header_page';
+import type { BadgeOptions } from '../../common/components/header_page/types';
+import { LandingPageComponent } from '../../common/components/landing_page';
+import { useLocalStorage } from '../../common/components/local_storage';
+import { SecuritySolutionPageWrapper } from '../../common/components/page_wrapper';
+import { DEFAULT_NUMBER_FORMAT } from '../../../common/constants';
+import { useSourcererDataView } from '../../common/containers/sourcerer';
+import {
+  useGetUserCasesPermissions,
+  useKibana,
+  useToasts,
+  useUiSetting$,
+} from '../../common/lib/kibana';
+import { SpyRoute } from '../../common/utils/route/spy_routes';
+import { useSignalIndex } from '../../detections/containers/detection_engine/alerts/use_signal_index';
+import * as i18n from './translations';
+
+const LOCAL_STORAGE_KEY = 'dataQualityDashboardLastChecked';
+
+const badgeOptions: BadgeOptions = {
+  beta: true,
+  text: i18n.BETA,
+};
+
+const comboBoxStyle: React.CSSProperties = {
+  width: '322px',
+};
+
+const FormControlLayout = styled(EuiFormControlLayout)`
+  max-width: 500px;
+  height: 42px;
+
+  .euiFormControlLayout__childrenWrapper {
+    overflow: visible;
+  }
+`;
+
+const Option = styled.div`
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  width: 100%;
+`;
+
+const OptionLabel = styled.span`
+  font-weight: bold;
+`;
+
+const options: EuiComboBoxOptionOption[] = [
+  {
```

Comment:
I'd recommend we do i18n for these labels as Index life cycle management translates them too.